sentry/sentry-symfony/0.7/etc/packages/prod/sentry.yaml

```diff
@@ -0,0 +1,3 @@
+sentry:
+    dsn: "%env(SENTRY_DSN)%"
+    release: "%env(APP_RELEASE)%"
```

Comment:
How is it used by Sentry? The name seems too generic. Can't we just use `APP_ENV` instead?

Reply:
This is the version of the application, not the environment. Should contain something like a git tag or commit hash.